apps/new-tab/src/Settings.tsx

```
@@ -797,20 +1236,903 @@ function SettingsTrigger({
+                            setBookmarksShown(
+                              JSON.stringify([
+                                ...currentBookmarksShown,
+                                bookmark,
+                              ])
+                            );
+                          }
+                        }}
+                        class="font-medium text-black rounded-lg dark:text-white text-sm p-4 bg-[#EDECEB]
+                          border-1 border-[#C0C0B8] h-[30px] flex justify-center items-center gap-2
+                          dark:bg-[#121314] dark:border-[#3F3F47]"
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)